ci/autoscaler/pipeline.yml

```diff
@@ -4,24 +4,18 @@ anchors:
   acceptance-main-env: &acceptance-main-env
     SYSTEM_DOMAIN: "autoscaler.ci.cloudfoundry.org"
     DEPLOYMENT_NAME: "acceptance-main"
-    SERVICE_NAME: "autoscaler-acceptance-main"
-    SERVICE_OFFERING_ENABLED: true
     SKIP_SSL_VALIDATION: true
     NAME_PREFIX: "acceptance-main"
 
   acceptance-main-log-cache-env: &acceptance-main-log-cache-env
     SYSTEM_DOMAIN: "autoscaler.ci.cloudfoundry.org"
     DEPLOYMENT_NAME: "acceptance-lc-main"
-    SERVICE_NAME: "autoscaler-acceptance-lc-main"
-    SERVICE_OFFERING_ENABLED: true
```

Comment:
this has to be applied ... it will not work otherwise.

Reply:
Removing the SERVICE_NAME should work, as we use the enable_base_deployment to set the SERVICE_NAME to the deployment name in all ci script.